EthOn.rdf

```diff
@@ -1904,18 +1904,18 @@
         </owl:annotatedTarget>
         <rdfs:comment>All create transactions are directed toward a Contract Account they create.</rdfs:comment>
     </owl:Axiom>
-
+    
 
 
     <!-- http://ethon.consensys.net/EthOnConcept -->
 
     <owl:Class rdf:about="http://ethon.consensys.net/EthOnConcept">
         <suggestedStringRepresentation xml:lang="en">EthonConcept</suggestedStringRepresentation>
-        <rdfs:comment xml:lang="en">Groups al EthOn concepts.</rdfs:comment>
+        <rdfs:comment xml:lang="en">The purpose of concepts in EthOn is to group a finite number of concrete instances of states and/or relationships between states under a single indefinitely bounded term.</rdfs:comment>
```

Comment:
Good point... my only concern is I don't think most people are familiar with what a **concept** is in regular language, let alone in OWL. My thought was to make the plain-language use clear first and then extrapolate to OWL--but maybe that is a bridge too far.

Nevertheless this definition:
"This is the root concept of all concepts that are original to the EthOn OWL ontology."
should work just fine!

To make the definition clearest, can we add the definition of a concept somewhere else, like in an rdf:seeAlso (from Wikipedia: "A concept is an abstract idea representing the type and fundamental characteristics of some set of objects.")